oslo_vmware/rw_handles.py

```diff
@@ -310,30 +310,44 @@ def __init__(self, session, lease, url, file_handle,
 
     def _log_progress(self, progress):
         """Log data transfer progress."""
-        if (progress == 100 or (progress - self._last_logged_progress >=
-                                MIN_PROGRESS_DIFF_TO_LOG)):
-            LOG.debug("Data transfer progress is %d%%.", progress)
-            self._last_logged_progress = progress
+        if 'bytes_read' in progress:
+            metric = progress["bytes_read"]
+        else:
+            metric = progress["bytes_written"]
```

Comment:
Why don't we just put "bytes" in there instead of "bytes_read" and "bytes_written"? We never seem to use those in another way.

Reply:
ok will do